src/dashboard/src/components/api/views.py

```diff
@@ -541,6 +522,8 @@ def fetch_levels_of_description_from_atom(request):
     else:
         return get_levels_of_description(request)
 
+# TODO should this have auth?
+@_api_endpoint(expected_methods=['GET', 'POST'])
 def path_metadata(request):
```

Comment:
This had no auth or verb restriction previously - needs to be tested to see if it still works with external consumers.

Also, it should return 'message': status and possibly 'error': False to match the rest of the API.